src/dune_rules/alias_rec.ml

```diff
@@ -0,0 +1,37 @@
+open Import
+
+include Action_builder.Alias_rec (struct
+  module Exists = struct
+    module T = struct
+      type t = Action_builder.alias_status
```

Comment:
It feels more natural for the monoid instance to come from `Alias_builder.Alias_status.t`.